package.json

```
@@ -25,7 +25,7 @@
   ],
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged && concurrently -n eslint,test,typescript 'npm run lint' 'npm test' 'npm run typescript'"
+      "pre-commit": "pretty-quick --staged && npm run lint && npm test && npm run typescript"
```

we need these to run concurrently, is there any reason why you removed it?

---

I think that the syntax we're using with concurrently might not work on windows, but this syntax definitely does work on windows:

```suggestion
      "pre-commit": "pretty-quick --staged && concurrently 'npm:lint' 'npm:test' 'npm:typescript'"
```

---

I think the concurrently command as it currently is might have troubles working on windows. To fix, you can do this:

```suggestion
      "pre-commit": "pretty-quick --staged && concurrently npm:test npm:lint npm:typescript"
```